tools/ARIAtools/ARIAProduct.py

```diff
@@ -182,6 +180,7 @@ def __mappingData__(self, file, rmdkeys, sdskeys):
         datalyr_dict={}
 
         # Setup rdrmetadata_dict
+        # for i, j in enumerate(rdrmetakeys):
```

Comment:
not sure this is needed here.

Reply:
removed i from the loop and the commented out line as well.